react/Icon.js

```diff
@@ -0,0 +1,25 @@
+import Check from './components/icon/Check/index'
+import Close from './components/icon/Close/index'
+import Deny from './components/icon/Deny/index'
+import Download from './components/icon/Download/index'
+import Edit from './components/icon/Edit/index'
+import Failure from './components/icon/Failure/index'
+import Plus from './components/icon/Plus/index'
+import Search from './components/icon/Search/index'
+import Success from './components/icon/Success/index'
+import Upload from './components/icon/Upload/index'
+import Warning from './components/icon/Warning/index'
+
+export default {
```

Comment:
Acabei de me dar conta que isso não vai funcionar, pois o `withHMR`, registra `null` no `__RENDER_7_COMPONENTS__` se o que vc está exportando não é um componente.

Vamos mudar esse comportamento lá então?

Reply:
PS: o `withHMR` sempre é usado, seja build dev ou prod, mas se não for `module.hot` ele simplesmente devolve seu componente puro.